python/core/auto_generated/qgsfeaturerequest.sip.in

```diff
@@ -736,6 +736,21 @@ For example, this should be set on requests that are issued from an
 expression function.
 
 .. versionadded:: 3.4
+%End
+
+    void iterateFidsOnly( bool useFids );
```

Comment:
In retrospect the name was badly chosen, I'll change the name. But the goal if this function was to trigger the other behavior. The reason being that if someone changes the filter from fids to expression or something else, they could specify if they want the stacking method or override the filters.

I'll change the name to 'stackFilters' instead, this should be less confusing.

Reply:
Let me know if this is clearer.

Reply:
Do you want to have stacking as a default behaviour without any safety measure? Personally I don't mind and it make sense that a if a users wants multiples filters he should set those filters and assume that they stack. But since the current behaviour was only to have one active filter (and the fiilter pass relies on the latest type of filter set.) This would mean that a user would have to set his expression filter first and then his ID filter. Otherwise setting the ID filter and then the expression would break things.

What can be done is to remove the variable tracking the latest filter type and simply check of the filters for each type are set during evaluation ( removing the need for a case check).

What is your suggested approach?